### PR TITLE
Fixes literal keywords in labels; Fixes CMDHELPER-2990 (null slot in arrays)

### PR DESCRIPTION
**Pull request by PseudoKnight**

### CODE REVIEW

**Comment by PseudoKnight** [comment]
I think I need to add some functionality to support array('null': @item) for set_pinv temporarily, so it doesn't silently break for people all over again.

File: src/main/java/com/laytonsmith/core/constructs/CDouble.java [hunk 1]
```diff
@@ -42,6 +42,6 @@ public CDouble clone() throws CloneNotSupportedException{
 
     @Override
     public boolean isDynamic() {
-        return true;
+        return false;
```
Comment (LadyCailin) [inline src/main/java/com/laytonsmith/core/constructs/CDouble.java]: What is the purpose of this change?
